jupyterhub/tests/mocking.py

```diff
@@ -246,6 +246,13 @@ def __init__(self, *args, **kwargs):
         if 'allow_all' not in self.config.Authenticator:
             self.config.Authenticator.allow_all = True
 
+        if 'api_url' not in self.config.ConfigurableHTTPProxy:
```

Comment:
Needed to avoid breaking
https://github.com/jupyterhub/jupyterhub/blob/718f01e600ba55a65618d820161b72073767a3f8/jupyterhub/tests/test_proxy.py#L38